frontend/src/components/InformationSlideshow/InformationSlideshow.jsx

```diff
@@ -14,17 +14,20 @@ export default function InformationSlideshow(props) {
     //usestates
     const [errors, setErrors] = useState({})
     const [firstResponse, setFirstReponse] = useState()
-    const [description, setDescription] = useState(["This is Paris", "Known as the city of love, it attracts millions of tourists a year",
-                                                        "Located in Western Europe, it is few hours away by plane","And be sure to fill up on French cuisine!" ])
-    const [image, setImage] = useState("")
-
+    const [description, setDescription] = useState(["Unfortunately, this location does not have information available. :("])
+    const [image, setImage] = useState("https://images.unsplash.com/photo-1584824486509-112e4181ff6b?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=870&q=80")
+    var [infoTitle, setInfoTitle] = useState("Information Slideshow!")
     // takes in information 
-    if (input === undefined){
+    
+    if (props.location === undefined){
         input = 0
+        console.log("info title", infoTitle)
```

Comment:
REQUIRED: Remove console logs in your commits. Use Javascript Console Utils (VSCode Extension) to easily make and remove them.